corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -365,7 +365,7 @@ class OwnershipCleanliness(models.Model):
     We use this field to optimize restores.
     """
     domain = models.CharField(max_length=100, db_index=True)
-    owner_id = models.CharField(max_length=100, db_index=True, primary_key=True)
+    owner_id = models.CharField(max_length=100, db_index=True)
```

Comment:
of course our data is messy and `owner_id`s span domains. this is sometimes true for random IDs because of test/api users, and especially true of owners like `demo_user` and `commtrack-system`